index.html

```diff
@@ -0,0 +1,177 @@
+
+              <hr class="hr-nav">
+              <span class="link"><a href="#speaker-section">Speakers</a></span>
+
+              <hr class="hr-nav">
+              <span class="link"><a href="#Sponsor">Sponsors</a></span>
+              <hr class="hr-nav">
+              <span class="link"><a href="#News">News</a></span>
+              <hr class="hr-nav">
+              <span class="link"><a href="#NFTintiative">NFT Initiative</a></span>
+              
+            </li>
+          </ul>
+        </div>
+
+        <div class="mobile-menu">
+          <i class="fa-solid fa-bars"></i>
+        </div>
+                  
+        <div class="first-section">
+          <h2 class="med-heading">
+            "Welcome to the Age of NFT's"
+          </h2>
+          <h1 class="main-heading">The 1st Annual <br> Nft Global Summit <br> 2022</h1>
+          <p class="paragraph1">An Expository event unravelling the Best Ideas, Best Builders, best Coverage and the Best community which covers over
+            100 countries taking place in July, in United States</p>
+          <h2 class="date">17(WED) - 29.07.2022(FRI)</h2>
+          <h4 class="venue">@ National Museum of National History, National Mall in Washington, D.C, United States</h4>
+        </div>
+      </section>
+
+      <!-- the Main Program Section -->
+      <section id="Program" class="program">
+        <div class="program-h2">
+          <h2>Main Program</h2>
+          <div class="line"> </div>
+        </div>
+        <div class="container">
+
+          <div class="card1 cards">
+            <i class="fa fa-desktop screen-icon" aria-hidden="true"></i>
+            <h3>Lecture</h3>
+            <p>Listen to the speakers from various countries about the Creating and selling NFTs</p>
+          </div>
+
+          <div class="card2 cards">
+            <i class="fa-solid fa-coins"></i>
+            <h3>NFT Exhibition</h3>
+            <p>Explore different NFTs already trending in the global market</p>
+          </div>
+
+          <div class="card3 cards">
+            <i class="fa-solid fa-comment-dollar"></i>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts and opinions with experts for each topic</p>
+          </div>
+
+          <div class="card4 cards">
+            <i class="fa-solid fa-chalkboard-user"></i>
+            <h3>Workshop</h3>
+            <p>try creating your own NFTs using our various open-end frameworks available</p>
+          </div>
+
+          <div class="card5 cards">
+            <i class="fa-solid fa-link"></i>
+            <h3>NFT Initate</h3>
+            <p>Get opportunities to link up and network with NFT traders and High class buyers all around the world,</p>
+          </div>
+
+        </div>
+        <h2 class="program-text desktop-only">View the Entire Program</h2>
+        <button type = "submit" class="program-text mobile-only">Join NFT Summit 2022</button>
+        
+      </section>
```

Comment:
Hi @mukhammadrizooff 
I STRONGLY disagree with this section of the review because:
 - the project requirements states that I can make changes like color and images but you are clearly requesting that I must change the program-headings to the color you stated.
 
IT MUST NOT BE EXACTLY
you are seriously trying to make my work exactly the same with the behance design which is not so in the project requirements.kindly re-review

I agree that I didn't add the -see the whole program text, and I will make changes to it

Reply:
## Hi Promise!
If you are not happy with the changes I requested. You can get a second opinion.
For that, you fill out the necessary form through [this link](https://airtable.com/shrQAqnBwek5a0O0s).
I wish you the best of luck in your endeavors.